decidim-core/app/commands/decidim/create_omniauth_registration.rb

```diff
@@ -59,6 +59,7 @@ def create_or_find_user
         @user.skip_confirmation! if verified_email
       end
 
+      @user.tos_accepted_at = @user.organization.tos_updated_at
```

Comment:
If I understand well, `@user.tos_accepted_at` is not the timestamp when the user accepted the tos but the version of the accepted tos. If this is the case, we should use another name like `@user.tos_version` or `@user.accepted_tos_version`.

Reply:
Yes you're right, `@user.tos_accepted_at` is the timestamp of the accepted version of the tos.
Maybe `@user.accepted_tos_updated_at` is more appropriate, as in [another PR](https://github.com/CodiTramuntana/decidim/pull/3) the organization will have the `tos_updated_at` attribute:

    add_column :decidim_organizations, :tos_updated_at, :datetime